### ISSUE TITLE
STRA for FAM

### ISSUE THREAD

**Issue by ConradBoydElliottGustafson** [bcgov/nr-forests-access-management#269]
As a MISO
I want to understand the secuirty risks of an app so that I can
so that make sure it follows ministry approved best practices and will not expose security risks

Acceptance Criteria:
MISO has completed STRA to review

**Definition of Done**
- [x] Ready to Demo in Sprint Review
- [x] Peer Reviewed by 2 people on the team
- [x] STRA has been submitted

Intake form:
https://apps.nrs.gov.bc.ca/int/confluence/download/attachments/83564991/Security%20Threat%20and%20Risk%20Assessment%20-%20Intake%20Form%20-%20NRM.docx?version=2&modificationDate=1612912602000&api=v2

Checklist:
https://apps.nrs.gov.bc.ca/int/confluence/download/attachments/83564991/LITE%20Application%20and%20Software%20Assessment%20-%20NRM.docx?version=2&modificationDate=1615933962000&api=v2

STRA docs at https://apps.nrs.gov.bc.ca/int/confluence/x/Q4OKCQ

**Comment by OlgaLiber2** [comment]
@gormless87 @ConradBoydElliottGustafson  should this be an epic? It's too large one ticket and we should break it down more. If we want to create a task I suggest something like "submit STRA for review" and link to all the dependencies (all the tickets and lead up to the submission). This way we can pull pieces of into the sprint and close the tickets as we complete the tasks.

**Comment by ConradBoydElliottGustafson** [comment]
It's from April 27, before we started working with Chris. I think this placeholder has been rendered redundant.

A new ticket "submit STRA for review" makes sense. I don't believe we have captured all the security work in tickets, so it's hard to retroactively pull them all into an epic at this point. I could be wrong!

**Comment by OlgaLiber2** [comment]
@gormless87 let's close this ticket and write a new one based on comments above.
